src/measure_theory/group.lean

```diff
@@ -0,0 +1,113 @@
+/-
+Copyright (c) 2020 Floris van Doorn. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Floris van Doorn
+-/
+import measure_theory.borel_space
+/-!
+# Measures on Groups
+
+We develop some properties of measures on (topological) groups
+
+* We define properties on measures: left and right invariant measures
+* We define the conjugate `A ↦ μ (A⁻¹)` of a measure `μ`, and show that it is right invariant iff
+  `μ` is left invariant
+-/
+noncomputable theory
+
+open has_inv set function
+
+namespace measure_theory
+
+variables {G : Type*}
+
+section
+
+variables [measurable_space G] [group G]
+
+/-- A measure `μ` on a topological group is left invariant if
+  for all measurable sets `s` and all `g`, we have `μ (gs) = μ s`,
+  where `gs` denotes the translate of `s` by left multiplication with `g`. -/
+def is_left_invariant (μ : set G → ennreal) : Prop :=
+∀ (g : G) {A : set G} (h : is_measurable A), μ ((λ h, g * h) ⁻¹' A) = μ A
```

Comment:
Could you please add a lemma `∀ {μ : measure G}, is_left_invariant μ <-> ∀ g, measure.map ((*) g) μ = μ`? Are you going to apply this definition to a function `set G → ennreal` other than a measure?

Reply:
It makes sense for outer measures, so that's why I defined it in this generality...

Reply:
added requested lemma